include/s3select_oper.h

```diff
@@ -2676,6 +2685,387 @@ class base_timestamp_to_string : public base_function
 
 };
 
+
+class base_like : public base_function
+{
+  protected:
+    value like_expr_val;
+    value escape_expr_val;
+    bool constant_state = false;
+    #if REGEX_HS
```

Comment:
upon `REGEX_HS` is set 
but HS is not installed (headers are missing)
Does the build fail?
same question for RE2